iOS/Flows/Conversation/ConversationViewControllerDataSource.swift

```diff
@@ -18,18 +18,25 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
     /// Model for the main section of the conversation or thread.
     /// The parent message is root message that has replies in a thread. It is nil for a conversation.
     struct SectionType: Hashable {
-        let cid: ConversationID
+        let sectionID: String
+        let conversationsController: ConversationListController?
         let parentMessageID: MessageId?
```

Comment:
This section type now accommodates conversationLists, conversations and threads.